deliveries/KZero_m2.md

```diff
@@ -0,0 +1,29 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account that submitted the application is allowed to submit milestones. 
+> 
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** [KZero Grant Application](https://github.com/w3f/Grants-Program/blob/master/applications/KZero.md)
+* **Milestone Number:** 2
+
+## KZero zk circuit implementation
+
+**Deliverables**
+Number	Deliverable	Specification
+| Number | Deliverable               | Specification                                                |
+| ------ | ------------------------- | ------------------------------------------------------------ |
+| 0a.    | License                   | GPLv3                                                        |
+| 0b.    | Documentation             | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| 0c.    | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
```

Comment:
@dejavukong could you share the link to the testing guide? Or is it identical to _[3. Docs](https://github.com/kzero-xyz/kzero-grant-docs/blob/main/kzero-integration-guide.md)_?

Reply:
@takahser The implementation of kzero is divided into two important modules: kzero-primitives and kzero-pallet. The testing instructions for each are listed separately in the testing sections of their respective documentation and are included in item 2. Tests of the deliverables.